### PR TITLE
module 2

### PR DESCRIPTION
**Pull request by homergit**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,22 +128,27 @@
...
+
+  <main id="main">
 
-  <div class="section" id="about">
+  <section id="about" class="section" id="about">
     <div class="section-heading has-text-centered">
-      <div class="title is-2">About Us</div>
-      <div class="subtitle is-5">We of all trades, master of "some"</div>
+      <h2 class="title is-2">About Us</h2>
+      <h5 class="subtitle is-5">We of all trades, master of "some"</h5>
```
Comment (IrinaShilina) [inline index.html]: You can't have `h5` right after `h2`. It should be `h3`. Run Lighthouse test to check.

File: index.html [hunk 2]
```diff
@@ -1,22 +1,74 @@
...
+  <title>Web Accessibility 2021</title>
   <link rel='stylesheet prefetch' href='https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.2/css/bulma.min.css'>
   <link rel='stylesheet prefetch' href='https://use.fontawesome.com/releases/v5.0.10/css/all.css'>
   <link rel="stylesheet prefetch" href="css/styles.css">
 </head>
 
 <body>
-  <div class="hero hero-bg is-fullheight">
+  <header class="hero hero-bg is-fullheight">
+    <div class="skip-navigation">
```
Comment (IrinaShilina) [inline index.html:18]: This section is usually visually hidden

File: index.html [hunk 3]
```diff
@@ -1065,71 +1127,73 @@ <h1 class="title is-size-4">Components</h1>
...
-          <a><i class="fab fa-youtube fa-2x"></i></a>
-          <a><i class="fab fa-facebook fa-2x"></i></a>
-          <a><i class="fab fa-twitter fa-2x"></i></a>
+          <a tabindex="0"><i class="fab fa-github-alt fa-2x"></i> <div class="visually-hidden">github</div></a>
+          <a tabindex="0"><i class="fab fa-youtube fa-2x"></i> <div class="visually-hidden">youtube</div></a>
+          <a tabindex="0"><i class="fab fa-facebook fa-2x"></i> <div class="visually-hidden">facebook</div></a>
+          <a tabindex="0"><i class="fab fa-twitter fa-2x"></i> <div class="visually-hidden">twitter</div></a>
         </div>
-        <p>The source code is licensed <a href="http://opensource.org/licenses/mit-license.php">MIT</a>.</p>
+        <p>The source code is licensed <a tabindex="0" class="third-party-resources" tabindex="0" href="http://opensource.org/licenses/mit-license.php" target="_blank"><i class="fas fa-external-link-alt"></i> MIT <div class="visually-hidden">will open in a new tab</div></a>.</p>
```
Comment (IrinaShilina) [inline index.html]: The "will open in a new tab" text should probably be inside the link itself. E.g. <a>MIT <span class="visually-hidden">will open in a new tab</a>

File: index.html [hunk 4]
```diff
@@ -684,31 +743,30 @@
...
+        <h5 class="subtitle is-5">Discuss any topic with no leaving this page</h5>
       </div>
 
       <div class="columns">
-        <div class="column is-3">
-          <a class="button is-primary is-block is-alt is-large">New Post</a>
+        <h3 class="column is-3">
+          <a tabindex="0" tabindex="0" class="button is-primary is-block is-alt is-large">New Post</a>
           <div class="menu">
             <ul class="menu-list forum-list">
```
Comment (IrinaShilina) [inline index.html:771]: Items inside the list should probably be links as well as they are the forum's tags.

File: index.html (hunk 1 above)
Comment (homergit) [reply index.html]: Done, thanks. I changed all h5 to h3 at headers of sections. Except for tiles.

File: index.html (hunk 2 above)
Comment (homergit) [reply index.html:18]: Fixed

File: index.html (hunk 3 above)
Comment (homergit) [reply index.html]: Resoled, thanks

File: index.html (hunk 4 above)
Comment (homergit) [reply index.html:771]: Done, I changed `span` with `a`

**Pull request merged**